<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSetting</class>
 <widget class="QDialog" name="DialogSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>处理模式</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxProcMode">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>87</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>PPP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FFT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RPP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DPRT-1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DPRT-2</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxPulseAccu">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>87</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>64</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>128</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>256</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>512</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1024</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>脉冲累积数</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxRangeAvg">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>87</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>距离平均</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxWidthAvg">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>87</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>谱平均</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxWinType">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>87</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Regular</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hamming</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Blackman</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Exact Blackman</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vnohan</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>窗口类型</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxBinNum">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>87</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>125</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>数据库长</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxRangeNorm">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>距离订正通断</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxR2Enable">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>R2使能</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxCMS">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>单库杂波消除</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxDsr">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>速度斑点消除</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxLsr">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>强度斑点消除</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox3x3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>3×3平滑输出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>LOG阀值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>SIG阀值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>CCOR阀值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>SQI阀值</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxLogThreshold">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxSigThreshold">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxCCORThreshold">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxSQIThreshold">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
